docs/First-Installation.md

```diff
@@ -113,49 +109,34 @@ For more information, see [Hotkeys](RetroArch-Configuration#hotkeys)
-
 ## Transferring ROMs
 
 You will not see any game systems (NES, n64, Playstation, etc) on the main menu until you add ROMs! Visit the [Transferring ROMs](Transferring-Roms) page to learn how to transfer ROMs to RetroPie.
 
-## AUDIO
+## Audio
 
-In general RetroPie audio will work out of the box without any tweaking, but if you have audio issues you should follow the instructions on the [Sound Issues Page](Sound-Issues) to fix them. You will most likely need to visit the [Sound Issues Page](Sound-Issues) if you are using a USB Audio device, or if you are using an aftermarket RPi HAT add-on audio device (such as a Justboom sound card).
+In general RetroPie audio will work out-of-the-box without any tweaking if using HDMI, but if you have audio issues you should follow the instructions on the [Sound Issues Page](Sound-Issues) to fix them. You will most likely need to visit this page if you are using a USB audio device, or if you are using an aftermarket RPi HAT add-on audio device (such as a Justboom sound card).
 
 ## PLAY!
 
-After you've added your roms you need to restart EmulationStation in order for them to show up. You can restart EmulationStation from the start menu, or by rebooting your pi with `sudo reboot`. 
+After you've transferred your ROMs, you need to restart EmulationStation in order for them to show up. You can restart EmulationStation by pressing Start > Quit > Restart EmulationStation, or with SSH access by rebooting your pi with `sudo reboot`. Once rebooted, you should see the game systems appear on the main menu alongside the original "RetroPie" menu.
+
+## Additional Setup Options
+
+* [Configure Wifi](Wifi) to allow for easy [Updating](Updating-RetroPie), downloading themes, and SSH
+* [Enable SSH](SSH) to connect to your Raspberry Pi remotely and make it easy for [Transferring ROMs](Transferring-Roms)
+* Configure more controllers. This can be done after plugging in the new controller and pressing *Start* on your controller and selecting *Configure Input*
+* Install additional emulators/cores. From the RetroPie main menu, select *RetroPie Setup > Manage Packages > Manage Optional Packages*. Stable additional packages are available here with more unstable packages listed in the *Manage experimental packages* section. The packages are ordered first by type (emulators / libretro cores / ports), then alphabetically. By selecting a package you can choose to install it or remove it. Some packages also have additional configurations.
+  * It is recommend to install emulators in the *RetroPie - libretrocores* section if you wish to retain hotkey functionality (saving states, rewinding, etc) as hotkeys will usually only work with these types of emulators. Libretro emulators have names that begin with "lr-", such as "lr-flycast" or "lr-mame2000".
```

Comment:
I don't agree with this - standalone emulators can have benefits and are occasionally the defaults for certain systems.

Reply:
But they're already installed. I believe it's important to make a note of this for people searching for additional emulators. For a while I thought my controller was broken because I couldn't open RGUI after switching emulator.

Reply:
What are already installed? the default installed emulators are a mix of libretro and standalone. We cannot give wholesale recommendations to libretro ones as it depends on your hardware (libretro cores are less performant). Besides, this is the wrong page to give recommendations of this nature IMO.

Reply:
The default _selected_ emulators are all Libretro, though. So someone who doesn't read this single paragraph is going to wonder why the emulator they just switched to / downloaded now can't fast-forward or open RGUI and think the controller needs to be reconfigured or have to ride the Google train to get answers - like I did. Plus, this paragraph isn't suggesting people use all Libretro cores, it's just explaining that some common features that they've been using will go missing if they switch. I'll get rid of it if you want me to, I just want to make sure you understand where I'm coming from as a new-ish RetroPie user. Your call.

Reply:
> The default selected emulators are all Libretro, though.

They're not - it depends on the hardware. pi0 uses pifba by default, for example.

> think the controller needs to be reconfigured

some standalone emulators are automatically configured.

in any case IMO it's the wrong page for advice on what emulators to download, and the subject is just too broad. individual system pages are the right place for emulator recommendations.